src/swig_python/python_extra.py_in

```diff
@@ -16,13 +16,19 @@ def _wrap_bin(fn, length, resize=False):
         return buf
     return wrapped
 
-def _wrap_int_array(fn, length):
+def _wrap_int_array(fn, length, resize=False):
     """ Wrap functions that take an output integer array to create/return it """
     def wrapped(*args):
         n = length(*args) if callable(length) else length
         out = [None] * n
         ret = fn(*list(args)+[out])
-        assert ret == len(out) # Input list must be of the correct size
+        if resize:
+            # Truncate as per _wrap_bin
+            assert ret <= n
+            return out[0:ret] if ret != n else out
+        if ret is not None:
```

Comment:
Why do we need this?

Reply:
There was a duplicated assert which we didn't need and I've just removed, good spotting.

The `None` check is needed for any future wrapper that returns a fixed size `uint32_t` array, but we want to wrap with a potential callable to determine its output size from its inputs.